tests/python/pants_test/backend/python/tasks2/BUILD

```diff
@@ -3,16 +3,30 @@
 
 
 python_tests(
-  name='select_interpreter',
-  sources=['test_select_interpreter.py'],
-  dependencies=[
-    '3rdparty/python:mock',
-    '3rdparty/python:pex',
-    'src/python/pants/backend/python:interpreter_cache',
-    'src/python/pants/backend/python:python_setup',
-    'src/python/pants/backend/python/targets',
-    'src/python/pants/backend/python/tasks2',
-    'src/python/pants/base:exceptions',
-    'tests/python/pants_test/tasks:task_test_base',
-  ]
+    name='resolve_requirements',
```

Comment:
Any reason not to just use default name and globs and leverage `py.test` passthrough, ie: `./pants test ... -- -k<pattern>`?